node/bin/src/main.rs

```diff
@@ -34,6 +34,15 @@ pub async fn main() {
     // ======= Run tasks ===========
     let main_stop = stop_receiver.clone(); // keep original for Prometheus
 
+    let _sentry_guard = if let Some(sentry_url) = config.general_config.sentry_url.clone() {
```

Comment:
intersting - here we use `if let Some()` but in other place we use compile feature... Seems inconsistent! What happens if we set the feature but forget to set this config (or vise versa)?

Reply:
The key point here that it's tracing that gets additional layer to work with sentry, but not the other way around, so that's why it's done as additional feature. If we are talking about scenario where we always will have only one binary running, I don't have problem with making sentry integration mandatory. I don't see much inconsistency because we are talking about separate crates:

Reply:
> one is the binary which we want to have both features
> second is a library, which(if used by some other binary) can include the feature or not

Oh, thanks for the explanation! It makes more sense now... Re: compilation flag vs config option it still feels like a flag is an overkill and we should just do an `Option`, but I acknowldge I may be wrong here. Curious what @itegulov thinks on this topic! But this shouldn't block this PR